### PR TITLE
Mark mask as not implemented

### PR DESCRIPTION
**Pull request by jasonrclark**
Addresses #527 

Currently can't find good way to do this in SWT so cutting it.

### CODE REVIEW

File: samples/simple_mask.rb [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-  animate 10 do
-    @stripes.clear do
-      20.times do |i|
-        strokewidth 4
-        stroke rgb(rand(0.0..0.5), rand(0.0..1.0), rand(0.0..0.3))
-        line 0, i * 5, 400, i * 8
-      end
-    end
-  end
-end
```
Comment (PragTob) [inline samples/simple_mask.rb]: could we maybe do a folder like "unsupported" or something, feels sad just deleting them :)

File: shoes-core/lib/shoes/dsl/art.rb [hunk 2]
```diff
@@ -251,6 +251,14 @@ def shape(*args, &blk)
...
         create Shoes::Shape, left, top, opts, blk
       end
+
+      def mask(*_)
+        raise Shoes::NotImplementedError,
+              <<~EOS
+                Sorry mask is not supported currently in Shoes 4!
+                Check out github issue #527 for any changes/updates or if you want to help :)
+              EOS
+      end
```
Comment (PragTob) [inline shoes-core/lib/shoes/dsl/art.rb]: :+1:

File: shoes-core/spec/shoes/dsl_spec.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+describe 'Unimplemented DSL methods' do
+  include_context "dsl app"
+
+  it 'video throws a Shoes::NotImplementedError' do
+    expect { dsl.video }.to raise_error Shoes::NotImplementedError
+  end
+
+  it 'mask throws a Shoes::NotImplementedError' do
+    expect { dsl.mask }.to raise_error Shoes::NotImplementedError
+  end
```
Comment (PragTob) [inline shoes-core/spec/shoes/dsl_spec.rb]: :+1:

File: samples/simple_mask.rb (hunk 1 above)
Comment (jasonrclark) [reply samples/simple_mask.rb]: Good idea!

**Comment by jasonrclark** [comment]
Added the new `samples/unsupported` directory and put both the `mask`, `video`, and `image` blocks samples into it.

**Pull request merged**